### DISCUSSION TITLE
Github Copilot

### DISCUSSION THREAD

**Discussion by rflmedina** [community/community#18211] [Copilot Conversations] [answered]
Is there a copilot release date?

**Comment by MrKrishnaAgarwal** [comment, marked as answer]
On 26 October 2021, GitHub Copilot was released as a plugin on the JetBrains marketplace. On 27 October 2021, GitHub released the GitHub Copilot Neovim plugin as a public repository. On 29 March 2022, GitHub officially announced Copilot's availability for the Visual Studio 2022 IDE.

Sign Up Here: https://github.com/features/copilot/signup

I hope it helps

it'll become generally available starting sometime this summer, Microsoft announced at Build 2022. Copilot will also be available free for students as well as “verified” open source contributors

**Comment by 3sv1n** [comment]
Can I write feedback and suggestions on GitHub Copilot here?

**Reply by MrKrishnaAgarwal** [reply]
Yes
